open-company/open-company-storage
# Public and private access in activity response

From [QA doc](https://paper.dropbox.com/doc/Post-beta-bugs-and-comments--AeKTdIEUmhMgJAbXXPP6Rxz7Ag-SPQWk76sbpqMUySRddhEm)

Item
> Add “(private)” or “(public)” after the section name in the email + Slack digest

Review with:
- [Digest](https://github.com/open-company/open-company-digest/pull/23)
- [Email](https://github.com/open-company/open-company-email/pull/51)
- [Bot](https://github.com/open-company/open-company-bot/pull/83)

To test:
- setup user A via Slack as admin
- setup user B as viewer
- add Section A as team access
- add Section B as private access with user B as contributor
- add Section C as public access
- add 1 post per section with user A
- send yourself an email digest
- [x] do you see `(private)` and `(public)` besides the section name only for secs B and C?
- send yourself a Slack digest
- [x] do you see `(private)` and `(public)` besides the section name only for secs B and C?
- now with user A and user B mark all the posts as unread
- with user A remind user B of the post in Section A
- [x] did you get an email? Has no access label besides the section?
- with user A remind user B of the post in Section C
- [x] did you get an email? Has `(public)` label besides the section?
- with user B remind user A of the post in Section A
- [x] did you get a Slack share? Has no access label besides the section?
- with user B remind user A of the post in Section B
- [x] did you get a Slack share? Has `(private)` label besides the section?
